src/main/java/interface_adapter/recipeCollection/RecipeCollectionState.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.recipeCollection;
+
+import java.util.List;
+
+/**
+ * The state information for the user's Recipe Collection.
+ */
+public class RecipeCollectionState {
+
+    private List<String> recipeList;
+
+    public void setRecipeList(List<String> recipeList) {
+        this.recipeList = recipeList;
+    }
+
```

Comment:
It would be not that informative but maybe include a JavaDoc here since you did it with another List getter just for consistency.